builtin/logical/aws/path_roles.go

```diff
@@ -301,6 +312,10 @@ func (b *backend) pathRolesWrite(ctx context.Context, req *logical.Request, d *f
 		roleEntry.IAMGroups = iamGroups.([]string)
 	}
 
+	if iamTags, ok := d.GetOk("iam_tags"); ok {
+		roleEntry.IAMTags = iamTags.(map[string]string)
```

Comment:
Will this panic if you pass in a JSON blob with non-string values?
```
{
...
  "iam_tags": {
    "tag1": 42
  }
}
```

Reply:
This approach works fine. It's apparent to me now that you can format a curl response in a number of ways, I may update the docs as part of this realization.